inbm/dispatcher-agent/dispatcher/schedule/sqlite_manager.py

```diff
@@ -41,7 +41,58 @@ def __init__(self, db_file=UDM_DB_FILE) -> None:
             raise DispatcherException(f"Error creating cursor: {e}")
         
         self._create_tables_if_not_exist() 
-            
+
+    def get_single_schedule_manifest(self) -> List[Tuple[int, int, int]]:
+        """
+        Get the data from single_schedule_manifest table and arrange them in ascending order based on the priority.
+        @return: List of tuple which represents priority, schedule_id and manifest_id.
+        """
+        try:
+            sql = ''' SELECT * FROM single_schedule_manifest; '''
+            self._cursor.execute(sql)
+            rows = self._cursor.fetchall()
+            data_list = []
+            for row in rows:
+                # Access the data using the column index or name
+                priority = row[0]
+                schedule_id = row[1]
+                manifest_id = row[2]
+                # Store the data in a tuple and append it to the list
+                data_tuple = (priority, schedule_id, manifest_id)
+                data_list.append(data_tuple)
+
+            # Sort the data_list by priority in ascending order
+            data_list = sorted(data_list, key=lambda x: x[0])
+            # Print the sorted data_list
+            return data_list
```

Comment:
I would recommend returning in a small dataclass so the caller can refer to the data by the column names rather than index, e.g.

```
@dataclass
class SingleScheduleManifest:
    priority: int
    schedule_id: int
    manifest_id: int
```

Reply:
Added.